plugins/image-prioritizer/tests/test-cases/common-lcp-image-and-lazy-loaded-image-outside-viewport-with-fully-populated-sample-data/set-up.php

```diff
@@ -31,31 +25,31 @@
 								'intersectionRatio' => 0.0, // Subsequent carousel slide.
 							),
 							array(
-								'xpath'             => '/HTML/BODY/DIV[@id=\'page\']/*[3][self::IMG]',
-								'isLCP'             => false,
-								'intersectionRatio' => 0 === $i ? 0.5 : 0.0, // Make sure that the _max_ intersection ratio is considered.
+								'xpath'              => '/HTML/BODY/DIV[@id=\'page\']/*[3][self::IMG]',
+								'isLCP'              => false,
+								'intersectionRatio'  => 0 === $i ? 0.5 : 0.0, // Make sure that the _max_ intersection ratio is considered.
+								'boundingClientRect' => array(
+									'width' => $viewport_width - 10,
+								),
 							),
 							// All are outside all initial viewports.
 							array(
 								'xpath'              => '/HTML/BODY/DIV[@id=\'page\']/*[5][self::IMG]',
 								'isLCP'              => false,
 								'intersectionRatio'  => 0.0,
-								'intersectionRect'   => $outside_viewport_rect,
```

Comment:
Note that the `intersectionRect` now automatically is populated to be zeroed out when the `interectionRatio` is 0. Previously it was not making sense to have a `DOMRect` containing non-zero values when the `intersectionRatio` was zero.